### DISCUSSION TITLE
Create a tag based on the file name

### DISCUSSION THREAD

**Discussion by Cleatus1972** [SilentVoid13/Templater#1276] [Templates Showcase]
Hello, I would like to automatically create a tag based on the file name. For example, the file name is "John Doe", I would like the tag to be #John_Doe. I would prefer not to put the underscore in the title name. Can someone help with this? I'm not a coder, so I apologize if this is too easy.

**Comment by Zachatoo** [comment]
It would look something like this

```js
#<% tp.file.title.replace(" ", "_") %>
```

**Comment by Cleatus1972** [comment]
Thanks! That worked
